.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
This file is clearly not related to your changes. Please remove it from the pull request.